js/currency.js

```diff
@@ -12,6 +12,21 @@ const zecLib = require("@missmonacoin/bitcoinjs-lib-zcash")
 const bchLib = require("@missmonacoin/bitcoincashjs-lib")
 const blkLib = require("@missmonacoin/blackcoinjs-lib")
 const jp = require('jsonpath')
+const { toCashAddress, toLegacyAddress, isLegacyAddress, isCashAddress }=require("bchaddrjs");
+
+if("function"!==typeof Uint8Array.prototype.slice0){
```

Comment:
Why is this override needed?

Reply:
See: https://github.com/bitcoincashjs/cashaddrjs/blob/master/src/cashaddr.js#L61    
`UInt8Array.prototype` is injected by someone and it causes an error.    
This code prevents it from errors and passes BCH address test.
![image](https://user-images.githubusercontent.com/10355528/42044497-26978e2e-7b34-11e8-944d-206c9668b379.png)

Reply:
Where is it overrided?
According to MDN slice() can have negative index.
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/slice

Reply:
Compare the screenshot.    
Left one is OK, because GitHub (and many websites) doesn't.    
Right one is not, since `UInt8Array.prototype` has `slice` with a link to declaration.